src/color/border.json

```diff
@@ -51,33 +51,39 @@
       },
       "disabled": {
         "onLight": {
-          "value": "{color.base.disabled.value}",
+          "value": "{color.base.gray.200.value}",
           "public": true,
           "onLight": true,
           "usage": "Disabled border color on light backgrounds",
           "wcag": "n/a"
         },
         "onDark": {
-          "value": "{color.base.disabled.value}",
+          "value": "{color.base.gray.200.value}",
           "public": true,
           "onDark": true,
           "usage": "Disabled border color on dark background",
           "wcag": "n/a"
         }
       },
       "Focus": {
-        "onLight": {
+        "newState": {
           "value": "{color.base.black-opacity-15.value}",
           "public": true,
+          "usage": "Focus border color for neutral backgrounds",
```

Comment:
what is a 'neutral' background?

Reply:
I am assuming that this is a reference to the new neutrals you added to base?

Reply:
Yes!